src/reference/asciidoc/logging.adoc

```diff
@@ -270,6 +270,42 @@ public class MyEnhancedAppender extends AmqpAppender {
 ----
 ====
 
+Starting with <BACKLOG VERSION>, the log4j2 `AmqpAppender` can be extended using `@PluginBuilderFactory` and extending also
+`AmqpAppender.Builder`
+
+
+.log4j2
+====
+[source, java]
+----
+public class MyEnhancedAppender extends AmqpAppender {
+
+    @Override
```

Comment:
It's not clear what kind of indents are here, but please, be sure that whitespaces (not tabs) are used in the docs for indents.

Reply:
OK, i will check my IDE